playwright-e2e/tests/rgp/adult-enrollment.spec.ts

```diff
@@ -162,15 +162,19 @@ test.describe.serial('Adult Self Enrollment', () => {
 
     const participantListPage = await navigation.selectFromStudy<ParticipantListPage>(Study.PARTICIPANT_LIST);
     await participantListPage.waitForReady();
+    const participantListTable = participantListPage.participantListTable;
 
-    await participantListPage.filterListByParticipantGUID(user.patient.participantGuid);
+    await expect(async () => {
+      await participantListPage.filterListByParticipantGUID(user.patient.participantGuid);
+      //Make sure the newly created participant can be found and it's participant page can be accessed
+      const participantListRowCount = await participantListTable.rowsCount;
+      expect(participantListRowCount).toBe(1);
+    }).toPass({
+      intervals: [10_000],
+      timeout: 120_000
+    });
```

Comment:
instead of checking just the once and failing if it doesn't see the guid - it repeatedly check every 10 seconds for about 2 mins - and then pass or fail